src/Process/PointCloudProcessing.js

```diff
@@ -212,78 +140,85 @@ export default {
     },
 
     update(context, layer, elt) {
-        const { shouldBeLoaded } = shouldDisplayNode(context, layer, elt);
+        elt.visible = false;
 
-        elt.shouldBeLoaded = shouldBeLoaded;
+        if (layer.octreeDepthLimit >= 0 && layer.octreeDepthLimit < elt.name.length) {
+            markForDeletion(elt);
+            return;
+        }
+
+        // pick the best bounding box
+        const bbox = (elt.tightbbox ? elt.tightbbox : elt.bbox);
+        elt.visible = context.camera.isBox3Visible(bbox, layer.object3d.matrixWorld);
+        if (!elt.visible) {
+            markForDeletion(elt);
+            return;
+        }
 
-        if (shouldBeLoaded > 0) {
-            elt.notVisibleSince = undefined;
+        elt.notVisibleSince = undefined;
 
-            // only load geometry if this elements has points
-            if (elt.numPoints > 0) {
-                if (elt.obj) {
-                    elt.obj.material.visible = true;
-                    if (__DEBUG__) {
-                        elt.obj.material.uniforms.density.value = elt.density;
+        // only load geometry if this elements has points
+        if (elt.numPoints > 0) {
+            if (elt.obj) {
+                elt.obj.material.visible = true;
+                elt.obj.material.uniforms.size.value = layer.pointSize;
 
-                        if (layer.bboxes.visible) {
-                            if (!elt.obj.boxHelper) {
-                                initBoundingBox(elt, layer);
-                            }
-                            elt.obj.boxHelper.visible = true;
-                            elt.obj.boxHelper.material.color.r = 1 - shouldBeLoaded;
-                            elt.obj.boxHelper.material.color.g = shouldBeLoaded;
+                if (__DEBUG__) {
+                    if (layer.bboxes.visible) {
+                        if (!elt.obj.boxHelper) {
+                            initBoundingBox(elt, layer);
                         }
+                        elt.obj.boxHelper.visible = true;
+                        elt.obj.boxHelper.material.color.r = 1 - elt.sse;
+                        elt.obj.boxHelper.material.color.g = elt.sse;
+                    }
+                }
+            } else if (!elt.promise) {
+                const distance = Math.max(0.001, bbox.distanceToPoint(context.camera.camera3D.position));
+                const priority = computeScreenSpaceError(context, layer, elt, distance) / distance;
+                elt.promise = context.scheduler.execute({
```

Comment:
Why do you divide by distance ?

Reply:
To load closest node first.

Reply:
If I understand correctly, minimizing SSE is what you want eventually (as things settle), but you prefer to bias the ordering of the fetches towards closer nodes, even if those have a smaller SSE. Is that correct ? I think a comment should be placed here for future reference (eg, we might want to enable the customization of this strategy)